platform/core/src/studies/services/wado/retrieveMetadata.js

```diff
@@ -468,37 +468,70 @@ async function createStudyFromSOPInstanceList(server, sopInstanceList) {
  * @param {string} studyInstanceUid The Study Instance UID of the study which needs to be loaded
  * @returns {Object} A study descriptor object
  */
-async function RetrieveMetadata(server, studyInstanceUid) {
-  return (server.enableStudyLazyLoad !== false
+async function RetrieveMetadata(server, studyInstanceUid, filters) {
```

Comment:
Update `JSDoc`, and what can be in `filters`?

Reply:
ok